Library/Homebrew/tap.rb

```diff
@@ -112,6 +112,16 @@ def to_hash
     }
   end
 
+  def load_tap_formula_renames
```

Comment:
There are several problems in this method. Since you store the information in a const `TAP_FORMULA_RENAME`, every time you load it, it will overwrite other taps' rename information. At the same, Ruby will generate warning as you are reassigning a const value. 

Also why do we have to store it in a ruby file? why not simple JSON which it's fast, reliable and more secure. 

So I would suggest you to implement it as:

```ruby
def formule_renames
  @formula_renames ||= if (rename_file = path/"formula_rename.json").file?
    Utils::JSON.load(rename_file.read)
  else
    {}
  end
end
```